src/trace_reader.rs

```diff
@@ -28,7 +28,7 @@ pub enum SampleType {
 
 #[allow(clippy::struct_field_names)]
 pub struct Sample {
-    pub location: Location,
+    pub locations: Vec<Location>,
```

Comment:
Is it a correct name? is it not callstack or path?

Current name suggested that this sample was collected from multiple places.